checks/__init__.py

```diff
@@ -392,17 +393,26 @@ def _get_check_manifest(self, manifest_path=None):
             CheckClass.__init__() -> AgentCheck.__init__() -> AgentCheck._get_check_manifest()
         """
         from checks.network_checks import NetworkCheck
+        try:
+            from checks.wmi_check import WinWMICheck
```

Comment:
We could also do if Platform.is_windows(), but generally we do the try/except import pattern, so I think it's better to follow the pattern we already have established